spanner/spanner_snippets/spanner/integration_test.go

```diff
@@ -573,6 +573,44 @@ func TestCreateDatabaseWithDefaultLeaderSample(t *testing.T) {
 	assertContains(t, out, "The result of the query to get")
 }
 
+func TestCustomInstanceConfigSample(t *testing.T) {
+	_ = testutil.SystemTest(t)
+	t.Parallel()
+
+	projectID := getSampleProjectId(t)
+	defer cleanupInstanceConfigs(projectID)
+
+	var b bytes.Buffer
+	baseConfig := fmt.Sprintf("projects/%s/instanceConfigs/%s", projectID, "nam11")
+	userConfig := fmt.Sprintf("custom-golang-samples-config-%v", randomID())
+	if err := createInstanceConfig(&b, "projects/"+projectID, userConfig, baseConfig); err != nil {
+		t.Errorf("failed to create instance configuration: %v", err)
```

Comment:
Use `t.Fatalf` here because subsequent operation(s) depend on this one succeeding?

Reply:
Thanks, done.